lib/swoosh/test_assertions.ex

```diff
@@ -63,6 +63,14 @@ defmodule Swoosh.TestAssertions do
   defp assert_equal(email, {:text_body, value}), do: assert(email.text_body == value)
   defp assert_equal(email, {:html_body, value}), do: assert(email.html_body == value)
 
+  defp assert_equal(email, {:regex, regex}) do
+    assert_equal(email, {:regex_html, regex})
+    assert_equal(email, {:regex_text, regex})
```

Comment:
You may not have both of them present

Reply:
Might be a better idea to extend the previous clause and simply match regex `value` and do regex compare.

Reply:
Done